src/ronin/profile/Profile.sol

```diff
@@ -449,6 +512,28 @@ contract Profile is IProfile, ProfileXComponents, Initializable {
     _setVRFKeyHash(_profile, vrfKeyHash);
   }
 
+  /**
+   * @inheritdoc IProfile
+   */
+  function changeSequencerAddr(address id, address sequencer) external onlyAdmin {
+    CandidateProfile storage _profile = _getId2ProfileHelper(id);
+
+    _requireNonZeroAndNonDuplicated(RoleAccess.SEQUENCER, sequencer);
```

Comment:
As the logic, aggregator address must be differential from sequencer address. Is it as intended?

Reply:
intended.